src/main/java/com/ldtteam/structurize/blockentities/interfaces/ISpecialCreativeHandler.java

```diff
@@ -0,0 +1,42 @@
+package com.ldtteam.structurize.blockentities.interfaces;
+
+import com.ldtteam.structurize.blueprints.v1.Blueprint;
+import com.ldtteam.structurize.placement.structure.AbstractStructureHandler;
+import com.ldtteam.structurize.util.PlacementSettings;
+import net.minecraft.core.BlockPos;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.world.level.Level;
+
+/**
+ * Special creative placement handler anchors.
+ */
+public interface ISpecialCreativeHandler
```

Comment:
Change the name to indicate that this needs to be implemented on an anchor block.